src/main/java/me/totalfreedom/totalfreedommod/FrontDoor.java

```diff
@@ -179,6 +179,10 @@ private Player getRandomPlayer(boolean allowDevs)
                 {
                     allowedPlayers.add(player);
                 }
+		if (!FUtil.FOUNDER.contains(player.getName()))
```

Comment:
The TFM Front Door is not designed like this and shouldnt be changed in this way.